_posts/2021-05-06-newsletter-9.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title:  "Imagining an Audience of Runners and Rylee's Run"
+date:   2021-05-06 07:00:29 -0500
+author: Andrew Battista
+categories: newsletters
+---
+
+One thing I have always liked about Rhetoric and Composition theory is the notion of an "imagined audience." The idea dates back to the 1970s and has been used as a device to help new writers approach the occasion of writing by forming a mental conception about the people who will be reading what they write. When [Walter J. Ong described it in 1975](https://www.jstor.org/stable/461344?seq=1#metadata_info_tab_contents), the primary example he used is a first year student in college who is assigned to write an essay about summer vacation experiences. Under no real world circumstances would there be an authentic occasion or reason for a student to share such information with a professor (or anyone, for that matter), nor would a professor actually be interested in it. Thus, in order to get the assignment finished, the student imagines an audience. It is a construct to communicate to those ostensibly interested in knowing about what happened during a summer vacation.
+
+The "imagined audience" construct helps me to think about writing, running, and writing about running. It is a way to conceptualize a challenge that many writers face. How does one address an audience that is a complete fiction, artifice that enables the process of writing in the first place? How does one engage an audience that is not a monolith, but instead is a complex group who may at first glance be united by a general shared interest but in reality exist on a wide continuum experiences, tastes, and abilities? Finally, what compels people (ourselves included) to write about running in the first place? It is possible to run without writing about it, just like it [is possible to run a marathon without posting a picture of it on social media](https://www.newyorker.com/cartoons/daily-cartoon/thursday-november-9th-marathon-forest).
```

Comment:
This one sentence needs attention. The second clause trips me up: "artifice that enables."